README.md

```diff
@@ -2098,14 +2115,7 @@ set :protection, :except => [:path_traversal, :session_hijacking]
 ```
 
 By default, Sinatra will only set up session based protection if `:sessions`
-has been enabled. Sometimes you want to set up sessions on your own, though. In
-that case you can get it to set up session based protections by passing the
-`:session` option:
-
-```ruby
-use Rack::Session::Pool
-set :protection, :session => true
-```
+have been enabled. See 'Using Sessions'.
```

Comment:
It could be assuming that `enable :sessions` was already called, but I can't tell.

If there is no equivalent test, then we broke the unspoken spec that is our README :(
When I say "we" it was probably me.